libs/shared/cms/src/lib/queries/iap-offerings-by-storeids/types.ts

```diff
@@ -0,0 +1,29 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import type { Enum_Iap_Interval } from '../../../__generated__/graphql';
```

Comment:
Left a comment about this in the Strapi repo

Reply:
Done!